code/modules/mob/living/simple_animal/friendly/catslug.dm

```diff
@@ -16,3 +16,60 @@
 	response_disarm = "rudely paps"
 	response_harm   = "kicks"
 	gold_core_spawnable = FRIENDLY_SPAWN
+
+	attack_vis_effect = ATTACK_EFFECT_SLASH
+	obj_damage = 0
+	melee_damage_lower = 0
+	melee_damage_upper = 0
+	attacktext = "stabs"
+	var/obj/item/twohanded/spear/weapon
+
+/mob/living/simple_animal/pet/catslug/UnarmedAttack(atom/A)
+	. = ..()
+	if(!isitem(A))
+		return
+	
+	if(!weapon && istype(A, /obj/item/twohanded/spear))
+		visible_message(span_notice("[src] wields the [A]."), span_notice("You wield the [A]."))
+		weapon = A
+		weapon.forceMove(src)
+		melee_damage_lower = weapon.force + weapon.force_wielded
+		melee_damage_upper = weapon.force + weapon.force_wielded
+		armour_penetration = weapon.armour_penetration
+		melee_damage_type = weapon.damtype
+		sharpness = weapon.sharpness
+		attack_sound = weapon.hitsound
+		update_icons()
+	else if(!weapon && !istype(A, /obj/item/twohanded/spear))
+		to_chat(src, span_warning("You do not know how to wield the [A]!"))
```

Comment:
```suggestion
	if(weapon)
		return
	if(istype(A, /obj/item/twohanded/spear))
		visible_message(span_notice("[src] wields the [A]."), span_notice("You wield the [A]."))
		weapon = A
		weapon.forceMove(src)
		melee_damage_lower = weapon.force + weapon.force_wielded
		melee_damage_upper = weapon.force + weapon.force_wielded
		armour_penetration = weapon.armour_penetration
		melee_damage_type = weapon.damtype
		sharpness = weapon.sharpness
		attack_sound = weapon.hitsound
		update_icons()
	else
		to_chat(src, span_warning("You do not know how to wield the [A]!"))
```

Reply:
do the messages look fine? we don't want the the here

Reply:
yes